donate/templates/pages/core/ways_to_give_page.html

```diff
@@ -0,0 +1,94 @@
+{% extends "pages/base_page.html" %}
+{% load i18n %}
+
+{% block title %}{% trans "Ways to Give" %}{% endblock %}
+
+{% block template_name %}app--ways-to-give{% endblock %}
+
+{% block header %}
+<div class="app__header">
+    <h1>{% trans "Ways to Give" %}</h1>
+    <div><a href="{{ways_to_give_link}}" class="button button--outline button--dark-bg">{% trans "donate now" %}</a></div>
```

Comment:
Let’s reuse the translation from another button + let localizers decide on uppercase usage

```suggestion
    <div><a href="{{ways_to_give_link}}" class="button button--outline button--dark-bg">{% trans "DONATE NOW" %}</a></div>
```